docs/smart_contracts.rst

```diff
@@ -0,0 +1,142 @@
+Raiden Network Smart Contracts
```

Comment:
Related to my previous comment: Perhaps rename this to Smart Contract Deployment